Userland/Applications/SystemMonitor/main.cpp

```diff
@@ -220,8 +227,11 @@ int main(int argc, char** argv)
     process_table_view.set_key_column_and_sort_order(ProcessModel::Column::CPU, GUI::SortOrder::Descending);
     process_model->update();
 
+    Config::pledge_domains("SystemMonitor");
```

Comment:
I would recommend moving this pledge a bit earlier in this main function (e.g. directly after the `unveil` calls), since if someone would want to read a config value before this pledge they would be allowed to read from any domains.

Reply:
We typically do this pledge right after constructing the application, FYI, since it needs to have an event loop.

Reply:
I moved it right after the unveils.